docs/commands/alias.md

```diff
@@ -54,23 +54,23 @@ flags:
 Aliases are most useful when they are persistent. For that, add them to your startup config:
 
 ```shell
-> config --set [startup ["alias myecho [msg] { echo $msg }"]]
+> config set [startup ["alias myecho [msg] { echo $msg }"]]
```

Comment:
This also needs an update to be: `config set startup ["alias myecho [msg] { echo $msg }"]`

Reply:
Actually, let me see if I can just make the tweak and get this landed.

Reply:
:+1: